### PR TITLE
PD-574 Standard-IA

### PR DESCRIPTION
**Pull request by nathan-px**

### CODE REVIEW

File: content/reference/CLI/credentials.md [hunk 1]
```diff
@@ -121,20 +123,32 @@ You can create and configure credentials for AWS by storing your secret access k
...
+    --use-iam s3cred \
+    <NAME>
     ```
     ```output
     Credentials created successfully, UUID:12345678-a901-2bc3-4d56-7890ef1d23ab
     ```
 
 ### Create credentials on AWS using IAM
 
+<!-- Does --s3-endpoint, s3-storage-class apply to these sections as well? -->
```
Comment (nathan-px) [inline content/reference/CLI/credentials.md:150]: @talakad Does --s3-endpoint, s3-storage-class apply to these sections as well?

Comment (talakad) [reply content/reference/CLI/credentials.md:150]: for IAM, end point does not apply. However storage class applies.

**Comment by nathan-px** [comment]
> looks good to me. These chanegs are applicable only for 2.5.3 onwards.. should that be mentioned somewhere? or the docs are only applicable for 2.5.3?

I typically don't note that unless it becomes problematic. We don't create new doc branches for subversions, and I can't add a note for every feature that's introduced with a subversion, that would be most of our documentation.

File: content/reference/CLI/credentials.md [hunk 2]
```diff
@@ -121,13 +127,23 @@ You can create and configure credentials for AWS by storing your secret access k
...
+    * The `--s3-endpoint` flag with the name of the endpoint (`s3.amazonaws.com`)
+    * The optional `--s3-storage-class` flag with either the `STANDARD` or `STANDARD-IA` value, depending on which storage class you prefer
+    * The `use-iam` flag with your s3 credentials
+    * The name of your cloud credentials
 
     ```text
     ./pxctl credentials create \
     --provider s3 \
     --s3-region us-east-1 \
-    --use-iam s3cred
```
Comment (nathan-px) [inline content/reference/CLI/credentials.md:130]: @talakad @disrani-px This step involves storing the credentials in env vars, is the `--use-iam` supposed to be here?

File: content/reference/CLI/credentials.md [hunk 3]
```diff
@@ -121,13 +127,23 @@ You can create and configure credentials for AWS by storing your secret access k
...
+    --s3-endpoint s3.amazonaws.com \
+    --s3-storage-class STANDARD \
+    --use-iam <s3-credentials> \
```
Comment (disrani-px) [inline content/reference/CLI/credentials.md:145]: The use-iam flag doesn't take a value. You don't need any creds with that flag

File: content/reference/CLI/credentials.md [hunk 4]
```diff
@@ -121,13 +127,23 @@ You can create and configure credentials for AWS by storing your secret access k
...
     AWS_ACCESS_KEY_ID=yyy
     ```
-2. Enter the `pxctl credentials create` command:
+2. Enter the `pxctl credentials create` command, specifying:
+
+    * The `--provider` flag with the name of the cloud provider (`s3`).
+    * The `--s3-region` flag with the name of the S3 region (`us-east-1`)
+    * The `--s3-endpoint` flag with the name of the endpoint (`s3.amazonaws.com`)
+    * The optional `--s3-storage-class` flag with either the `STANDARD` or `STANDARD-IA` value, depending on which storage class you prefer
+    * The `use-iam` flag with your s3 credentials
```
Comment (nathan-px) [inline content/reference/CLI/credentials.md:136]:
```suggestion
    * The `use-iam` flag
```

File: content/reference/CLI/credentials.md (hunk 3 above)
Comment (nathan-px) [inline content/reference/CLI/credentials.md:145]:
```suggestion
    --use-iam \
```

File: content/reference/CLI/credentials.md [hunk 5]
```diff
@@ -168,10 +184,15 @@ Instead of storing your secret access key and access key ID on the host, you can
     ```
 
 
-2. Enter the following pxctl credentials create command, specifying the S3 region for the account:
+2. Enter the following pxctl credentials create command, specifying the following:
+
+    * The `--provider` flag with the name of the cloud provider (`s3`).
+    * The `--s3-region` flag with the the S3 region associated with your account
+    * The optional `--s3-storage-class` flag with either the `STANDARD` or `STANDARD-IA` value, depending on which storage class you prefer
+    * The `use-iam` flag with your s3 credentials
```
Comment (nathan-px) [inline content/reference/CLI/credentials.md:192]:
```suggestion
    * The `use-iam` flag
```

File: content/reference/CLI/credentials.md [hunk 6]
```diff
@@ -207,10 +228,16 @@ Instead of storing your secret access key and access key ID on the host, you can
...
     ```
 
-2. Enter the following pxctl credentials create command, specifying the bucket's S3 region and bucket name:
+2. Enter the following pxctl credentials create command, specifying the following:
+
+    * The `--provider` flag with the name of the cloud provider (`s3`)
+    * The `--s3-region` flag with your bucket's s3 region
+    * The optional `--s3-storage-class` flag with either the `STANDARD` or `STANDARD-IA` value, depending on which storage class you prefer
+    * The `--bucket` flag with your bucket's name
+    * The `use-iam` flag with your s3 credentials
```
Comment (nathan-px) [inline content/reference/CLI/credentials.md:237]:
```suggestion
    * The `use-iam` flag
```

**Pull request merged**